atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -156,6 +173,57 @@ public void tearDown() {
         puncherStore = null;
     }
 
+    private void waitToForceRefresh() throws InterruptedException {
+        Thread.sleep(2 * SMALL_REFRESH_MILLIS);
```

Comment:
Thread.sleep forces your unit tests (which are supposed to be quick) to take a minimum amount of time which can be longer than actually needed. Also, you can't necessarily guarantee that what you're expecting to happen after the time will actually happen (although you did pipe it _all_ the way through).

I'd recommend looking into Awaitility (https://github.com/awaitility/awaitility/wiki/Usage) if you can't express your tests / code without awaiting! (you might be able to with the refactor). Namely, Awaitility polls until some expected condition has occurred, and fails if it doesn't occur within a timeout.

Reply:
nit: for tests, we generally put private utility methods at the bottom of the class.
For prod classes, specific auxiliary methods go under the public method that uses it (or at the bottom of the class)